apps/nextjs/package.json

```diff
@@ -23,7 +23,8 @@
     "with-env": "dotenv -e ../../.env --",
     "aila": "tsx scripts/aila-cli.ts",
     "storybook": "dotenv -e ../../.env -- storybook dev -p 6006 --no-open",
-    "build-storybook": "dotenv -e ../../.env -- storybook build"
+    "build-storybook": "dotenv -e ../../.env -- storybook build",
+    "chromatic": "pnpm with-env npx chromatic"
```

Comment:
Is npx safe here? Should it be pnpm dlx?

Reply:
Good call. Thank you